Runtime/Scripts/Extensions/MPEG/Tools/ActionModule.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace GLTFast {
+    public interface MPEG_ActionEvent { }
+
+    public struct MPEG_ActionManipulateEvent: MPEG_ActionEvent
```

Comment:
Shouldn't the action hold a reference to the nodes that need to be manipulated as specified in the glltf document ?